src/clev2er/algorithms/seaice/alg_cs2_wave_discimination.py

```diff
@@ -0,0 +1,201 @@
+""" clev2er.algorithms.seaice.waveform_discrimination.py
```

Comment:
I think 
clev2er.algorithms.seaice.waveform_discrimination.py
needs correcting to the actual name